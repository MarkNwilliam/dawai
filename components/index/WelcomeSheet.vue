<template>
  <v-col cols="12">
    <v-row align="center" justify="center">
      <v-card color="transparent" flat>
        <v-img
          height="250"
          width="250"
          :aspect-ratio="16 / 9"
          src="/logo-1024-transparent.svg"
        ></v-img>
      </v-card>
    </v-row>
  </v-col>
</template>
