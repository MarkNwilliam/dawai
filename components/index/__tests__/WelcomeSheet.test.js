// Libraries
import Vuetify from 'vuetify'

// Component
import { mount, createLocalVue } from '@vue/test-utils'
import WelcomeSheet from '../WelcomeSheet'

// Utilities
const localVue = createLocalVue()

describe('WelcomeSheet.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should render the welcome sheet snapshot', (done) => {
    const wrapper = mount(WelcomeSheet, {
      localVue,
      vuetify
    })

    // With jest we can create snapshot files of the HTML output
    expect(wrapper.html()).toMatchSnapshot()

    done()
  })
})
