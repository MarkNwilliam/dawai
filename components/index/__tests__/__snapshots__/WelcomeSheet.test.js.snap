// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeSheet.vue should render the welcome sheet snapshot 1`] = `"<v-col cols=\\"12\\"><v-row align=\\"center\\" justify=\\"center\\"><v-card color=\\"transparent\\" flat=\\"\\"><v-img height=\\"250\\" width=\\"250\\" aspect-ratio=\\"1.7777777777777777\\" src=\\"/logo-1024-transparent.svg\\"></v-img></v-card></v-row></v-col>"`;
